core/modules/filters/qfp.js

```diff
@@ -0,0 +1,40 @@
+/*\
+title: $:/core/modules/filters/qfp.js
```

Comment:
I'm not a huge fan of acronymic filter operators. `hash` doesn't seem right because this is only one of a number of possibly useful hashing algorithms we might want to expose. So maybe `numhash`?

Reply:
yes, numhash sounds good!